helix-core/src/main/java/org/apache/helix/model/Message.java

```diff
@@ -115,6 +116,11 @@ public enum MessageState {
     UNPROCESSABLE // get exception when create handler
   }
 
+  public enum STRebalanceType {
+    LOAD_REBALANCE,
+    RECOVERY_REBALANCE
+  }
+
```

Comment:
Not an issue, just a thought. I think there is existing `RebalanceType` in a throttling related class. For widely used and fundamental enum like this, maybe it deserves a separate class and reduce duplication

Reply:
Yes, we have thought about reusing the RebalanceType in StateTransitionThrottleConfig class, but reusing it may cause confusion as this use case is not related to throttling. Separating it out may be an option, but the enum in StateTransitionThrottleConfig is widely used, and it will need a lot of code changes, overall it's not worthy to do it, so it is also not considered right now.

Reply:
Agree, feel free to close this one. Thanks